app/views/migration/_form.html.erb

```diff
@@ -54,14 +54,15 @@
             <%= wr.input(:dc_type) %>
             <%= wr.input(:dcterms_created, as: :date, html5: true, input_html: date_format_fallback_attributes) %>
             <%= wr.input(:dc_creator) %>
+            <%= wr.input(:edm_rights, as: :radio_buttons, collection: edm_rights_options, checked: wr.object&.edm_rights&.id, wrapper_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.web-resource-license' }) %>
           <% end %>
         <% end %>
 
         <%= agg.nested_fields_for(:edm_hasViews, wrapper_tag: :div) do |wr| %>
           <%= field_set_tag do %>
             <%= wr.remove_nested_fields_link(i18n.t('buttons.object.remove')) %>
-            <%= wr.input(:media, label: i18n.t('legends.each_object', number: ''), label_html: { class: 'sequenced_object' }, hint: i18n.t('hints.edm_hasView.media'), input_html: { accept: "#{EDM::WebResource.allowed_extensions}, #{EDM::WebResource.allowed_content_types}" }) %>
-            <%= wr.input(:media_cache, as: :hidden) %>
+            <%= wr.input(:media, label: i18n.t('legends.each_object', number: ''), label_html: { class: 'sequenced_object' }, hint: i18n.t('hints.edm_hasView.media'), input_html: { accept: "#{EDM::WebResource.allowed_extensions}, #{EDM::WebResource.allowed_content_types}", 'data-makes-required': "contribution_ore_aggregation_edm_hasViews___nested_field_for_replace_with_index___edm_rights" }) %>
```

Comment:
Single quote the strings for the values of the `data-makes-required` attributes.